js/exercise-00.js

```diff
@@ -0,0 +1,20 @@
+/* 
+   Create a constant variable of each of the following primitive types:
+   - String
+   - Number
+   - Boolean
+   - Null
+   - Undefined
+
+   Log out each variable to the terminal at the end.
+*/
+const string = "Manel";
+const number = 18;
+const boolean = true;
+const nulo = null;
+const Undefined = undefined;
```

Comment:
Use camelCase convention to name variables.

```suggestion
const undefinedValue = undefined;
```
FIX: name properly variable

FIX: add an empty line between declarations block and logs block